src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
```suggestion
public class Lottery {
```
замість наслідування краще створити об'кт типу ColorSupplier всередині класу Lottery таким чином ти можеш комбінувати багато класів, а з наслідуванням тільки один